src/index.ts

```diff
@@ -14,6 +14,7 @@ if (!process.env.OPENAI_API_KEY) {
 if (!process.env.GITHUB_TOKEN) {
   throw new Error("GITHUB_TOKEN is required");
 }
+console.log("Environment variables validated");
```

Comment:
Lowkey, this console.log is for dev vibes only and should probs be removed or changed to a logger before going live fr. We tryna keep it professional out here 💼.